ocp_resources/resource.py

```diff
@@ -385,6 +388,16 @@ def __init__(
         self.teardown = teardown
         self.timeout = timeout
         self.delete_timeout = delete_timeout
+        self.dry_run = dry_run
+        self.node_selector = node_selector
+        self.node_selector_labels = node_selector_labels
+        self.node_selector_spec = self._node_selector_spec_prepare()
+
+    def _node_selector_spec_prepare(self):
```

Comment:
please rename, have the verb first:
_prepare_node_selector_spec

Reply:
ack